publish-backend/tests/post/index.js

```diff
@@ -51,26 +51,34 @@ describe("post", () => {
       const responsePost = response.body.data.attributes;
 
       expect(responsePost.publishedAt).toEqual(
-        postToCreate.data.publishedAt.toISOString()
+        postToCreate.data.publishedAt.toISOString(),
       );
       expect(responsePost.scheduled_at).toEqual(
-        postToCreate.data.publishedAt.toISOString()
+        postToCreate.data.publishedAt.toISOString(),
       );
     });
 
-    it("should create post excluding publishedAt and scheduled_at for contributors", async () => {
+    it("should create post excluding restricted fields for contributors", async () => {
+      const postToCreateCopy = { ...postToCreate };
+      postToCreateCopy.data.code_injection_head =
+        "<script>alert('test')</script>";
+      postToCreateCopy.data.code_injection_foot =
+        "<script>alert('test')</script>";
+
       const response = await request(strapi.server.httpServer)
         .post("/api/posts")
         .set("Content-Type", "application/json")
         .set("Authorization", `Bearer ${contributorJWT}`)
-        .send(JSON.stringify(postToCreate));
+        .send(JSON.stringify(postToCreateCopy));
 
       expect(response.status).toBe(200);
       const responsePost = response.body.data.attributes;
 
       // Should not set publishedAt and scheduled_at for contributors
       expect(responsePost.publishedAt).toBeNull();
       expect(responsePost.scheduled_at).toBeNull();
+      expect(responsePost.code_injection_head).toBeUndefined();
+      expect(responsePost.code_injection_foot).toBeUndefined();
```

Comment:
```suggestion
      expect(responsePost.codeinjection_head).toBeUndefined();
      expect(responsePost.codeinjection_foot).toBeUndefined();
```

To also check if this should be using `toBeNull`?